### ISSUE TITLE
what to set NGINX_SERVER_NAME to?

### ISSUE THREAD

**Issue by dwightmulcahy** [piku/deploy-gitea#2]
> piku config:set NGINX_SERVER_NAME=gitea.somedomain.net
> ...
> NGINX_SERVER_NAME is the domain name you want for the gitea server (the domain must point at the machine already).

I'm unfamiliar with gitea's setup so I don't know what to set this too... maybe expand the example a bit?

**Comment by chr15m** [comment]
@dwightmulcahy this is a standard Piku thing. Setting `NGINX_SERVER_NAME` instructs Piku to point that domain at the app. So if you're just using gitea internally you don't need to set this. If you want it to be available on the open web with a domain name then you set it (the machine must be on a public IP in that case obviously).

I will update the documentation to clarify this.

**Comment by dwightmulcahy** [comment]
Yeah I figured that out later and forgot to add to this.  Assume that people don't know what it is and document it.
